homeassistant/components/tradfri/switch.py

```diff
@@ -1,16 +1,11 @@
 """Support for IKEA Tradfri switches."""
-import logging
-
-from pytradfri.error import PytradfriError
-
 from homeassistant.components.switch import SwitchDevice
-from homeassistant.core import callback
-from . import DOMAIN as TRADFRI_DOMAIN, KEY_API, KEY_GATEWAY
+from . import KEY_API, KEY_GATEWAY
+from .base_class import TradfriBaseDevice
 from .const import CONF_GATEWAY_ID
 
-_LOGGER = logging.getLogger(__name__)
 
-TRADFRI_SWITCH_MANAGER = "Tradfri Switch Manager"
+TRADFRI_device_MANAGER = "Tradfri Switch Manager"
```

Comment:
I don't understand what this is for. Does this have any functionality in Home Assistant's inner workings?

Reply:
No, it doesn't seem so. We can remove it.

Reply:
OK, I removed it